src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/ParseRequestServiceTest.java

```diff
@@ -133,126 +159,110 @@ void validateUserProfiles_throwsResourceNotFound_someProfilesNotFoundTest() {
 
     @Test
     void validateJudicialProfilesTest() {
+        JudicialProfile judicialProfile2;
+        judicialProfile2 = judicialProfile;
+        judicialProfile2.setElinkId("37396");
```

Comment:
If you are dealing with more than one profile then you should create the new profile  for judicialProfile2 through the json file rather than assign the reference of judicialProfile. Because, both are pointing the same object in JVM heap memory whatever operation you will perform on judicialProfile2 it will impact on judicialProfile.

Reply:
I have actually now removed the second profile, as after looking at the results to the tests, it turns out that second profile was having no effect on the coverage, and so was redundant.